### DISCUSSION TITLE
Delete last word in search query

### DISCUSSION THREAD

**Discussion by rappie** [ibhagwan/fzf-lua#1323] [Q&A] [answered]
I would like to be able to delete the last word of the query I'm typing

Example:
I'm searching for testing deposits and type "test depos", then I realize that I want to search for withdrawals instead. Now I'd like to be able to easily remove "depos". In Emacs I'm used to doing Ctrl+Backspace, and in Telescope I can go to normal mode and for example do `bC`.

I'm quite new to Neovim, and I'm sure there are multiple solutions here. What do you recommend?

I saw this solution: https://github.com/ibhagwan/fzf-lua/discussions/1144, but these are two pretty complex keybinds, I'd rather have something simple.

Thanks!

**Comment by ibhagwan** [comment, marked as answer]
Fzf comes with default binds to help with this:
```
           unix-line-discard            ctrl-u
           unix-word-rubout             ctrl-w
```

ctrl-w will delete the last word, ctrl-u deletes the entire line, there is also alt-b to go backwards without deleting, etc, see `man fzf` for the complete list of default binds.

Fzf-lua matches these defaults but you can also modify these if you wish by setting new binds under `keymap.fzf`.

**Reply by rappie** [reply]
Perfect, thank you!

It never occurred to me to look at fzf's keybinds :slightly_smiling_face:
